migration/sql-files/025-fix-feature-level.sql

```diff
@@ -0,0 +1 @@
+UPDATE users SET feature_level = 'released' WHERE feature_level = '' and email like '%+preview%';
```

Comment:
what about adding a default value to the `feature_level` column as well ?

Reply:
It already has a default value, right ?